packages/mobile/src/send/utils.ts

```diff
@@ -14,6 +14,8 @@ import { PaymentInfo } from 'src/send/reducers'
 import { TransactionDataInput } from 'src/send/SendAmount'
 import { timeDeltaInHours } from 'src/utils/time'
 
+const DISMISS_SPEND_LIMIT_BANNER_TIMER = 5000
```

Comment:
Dupe of https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/config.ts#L38